client/src/less/bhima-bootstrap.less

```diff
@@ -8,6 +8,44 @@
 
 @border-radius-base: 2px;
 
+@ima-green:           #77a13f;
+@ima-dark-orange:     #e57b14;
+@ima-dark-blue:       #085484;
+@ima-light-orange:    #fdba64;
+@ima-light-blue:      #6fbdee;
+  
+/** @todo move all structure code into less with configurable variables */
+@expanded-offset:  260px;
```

Comment:
I'm guessing this is referring to the tree navigation?  If so, it might be helpful to either name the variables to reflect that, or leave a comment about it.